doc/nrf/app_dev/device_guides/nrf70/index.rst

```diff
@@ -115,6 +114,10 @@ The following nRF70 Series shields are available and defined in the :file:`nrf/b
      - ``nrf7002eb``, ``nrf7002eb_interposer_p1`` (nRF54 Series)
      - | :ref:`Development guide <ug_nrf7002eb_gs>`
        | `User Guide <nRF7002 EB User Guide_>`_
+   * - nRF7002 :term:`Expansion Board (EB)` II
+     - PCA63571
+     - ``nrf7002eb2`` (nRF54 Series)
```

Comment:
maybe add `(nRF54 Series, supersedes nRF7002eb + nRF7002EB interposer combo)` and `deprecated` to the above column?

Reply:
OK, I will add deprecate but interposer was not a publicly available board, so will avoid mentioning it.